lib/rollbax/logger.ex

```diff
@@ -69,7 +69,7 @@ defmodule Rollbax.Logger do
   end
 
   def handle_event({level, _gl, event}, %{metadata: keys} = state) do
-    if proceed?(event) and meet_level?(level, state.level) do
+    if proceed?(event) and meet_level?(level, state.level) and !whitelisted?(state[:whitelist], event) do
```

Comment:
Don't use `!` with strictly boolean expressions. Let's use `not whitelisted?(...)` here.